### PR TITLE
Feature/swed 2140 enzyme react testing library migration

### PR DESCRIPTION
**Pull request by goldenraphti**
Migrating tests from Enzyme to React Test Library, in order to enable React migration to 18 and onwards.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Review instructions
[Review instructions](../REVIEW_INSTRUCTIONS.md)

### CODE REVIEW

File: src/App/components/Tabs/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const Tabs = ({ id, scroll, items, ulId }) => {
             <ul id={ulId}>
                 {items.map((name, i) => (
                     <li key={`tab-item-${name}-${i}`} className={activeIndex === i ? "active" : null}>{"\n"}
-                        <a href="#" onClick={e => setActive(e, i)}>{name}</a>{"\n"}
+                        <a className="testing-clicks" href="#" onClick={e => setActive(e, i)}>{name}</a>{"\n"}
```
Comment (goldenraphti) [inline src/App/components/Tabs/index.js:18]: remove new class

**Review by eskilsen — DISMISSED** [review]
Holy moly, we are goooood 🔥 We gon' need a 🚒 firetruck 🚒  to stop us! WE ON FIRE 🔥

File: src/App/components/Alert/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ const Alert = ({ id, type, icon, close, text, className }) => {
 
 Alert.propTypes = {
     id: PropTypes.string,
-    type: PropTypes.oneOf(["success", "warning", "danger", "informative"]).isRequired,
+    type: PropTypes.oneOf(["success", "warning", "danger", "informative", "test"]).isRequired,
```
Comment (goldenraphti) [inline src/App/components/Alert/index.js:27]: remove test type

**Review by eskilsen — APPROVED** [review]
CALL THE FIRE DEPARTMENT! Two guys are on 🔥 !!

**Pull request merged**